tasks/jsinspect.js

```diff
@@ -24,9 +25,21 @@ module.exports = function(grunt) {
       identifiers: false,
       failOnMatch: true,
       suppress:    100,
-      reporter:    'default'
+      reporter:    'default',
+      configFile:  '.jsinspectrc'
     });
 
+    if (fs.existsSync(options.configFile) && fs.lstatSync(options.configFile).isFile()) {
+      var contents = strip(fs.readFileSync(options.configFile, 'utf8'));
```

Comment:
`0.5.0` or similar it should be and since we are in `0.x` mode, that should be "expected".
Perhaps we can make soon `1.0.0`...